packages/mirrorful/editor/src/components/ColorPalette/AddColorSkeleton.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+    Box,
+    Button,
+    Stack,
+    Text,
+    useDisclosure,
+    Badge,
+    Heading,
+    Divider,
+    Menu,
+    MenuButton,
+    MenuList,
+    MenuItem,
+    MenuItemOption,
+    MenuGroup,
+    MenuOptionGroup,
+    MenuDivider,
+    IconButton,
+    Icon,
+  } from '@chakra-ui/react'
+  import tinycolor from 'tinycolor2'
+  import { generateDefaultColorShades, handleInvalidColor } from './utils'
+  import { TColorData, TColorVariant } from 'types'
+
+  // Accepts: backgroundColorString, a valid CSS color string, eg '#EEE' or 'steel-blue', which sets the color of row elements.  An invalid color results in random assignment
+export function AddColorSkeleton ({ backgroundColorString, numberOfMockVariants }: { backgroundColorString: string, numberOfMockVariants: number } ) {
+
+    // Check to ensure input color is valid, adjust if not.
+    backgroundColorString = handleInvalidColor(backgroundColorString)
+
+    const colorScale = generateDefaultColorShades(backgroundColorString)
```

Comment:
could just pass in a default grey... probably the default chakra gray.500: https://chakra-ui.com/docs/styled-system/theme#colors